entity-framework/index.yml

```diff
@@ -90,7 +92,7 @@ additionalContent:
             text: "Blazor Server"
           - url: https://docs.microsoft.com/ef/ef6/fundamentals/databinding/wpf
             text: "Windows Presentation Foundation (WPF)"
-          - url: https://docs.microsoft.com/ef/ef6/fundamentals/databinding/winforms
+          - url: core/get-started/wpf/
```

Comment:
Even though this is URL, it is file path so we need to write `core/get-started/wpf.md`
Same for core/get-started/xamarin
Also apply same to change at line 61